winter-website-things-ive-made.css

```diff
@@ -25,7 +25,89 @@ h2.thingsIveMadeDetails {
 
 img, h3 {
 	width: 50%;
+	padding: 10px;
+	display:block;
```

Comment:
in your typing you didn't leave a space after the colon, all your code above you have been doing that, so you could keep this consistent pattern, because is easier to read